src/node_dns.h

```diff
@@ -0,0 +1,51 @@
+#ifndef SRC_NODE_DNS_H_
+#define SRC_NODE_DNS_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "../deps/getdns/build/getdns/getdns_ext_libuv.h"
```

Comment:
Our other dependencies’ headers end up in the include path, can we make that happen here as well?

Reply:
Yeah that's part of the "actually make this work with our build system" stage that I haven't gotten to yet :P